python/jupyterlab_widgets/jupyterlab_widgets/__init__.py

```diff
@@ -5,16 +5,11 @@
 
 
 def _jupyter_labextension_paths():
-    import sys
-    from pathlib import Path
-
-    labext_name = '@jupyter-widgets/jupyterlab-manager'
-    here = Path(__file__).parent.resolve()
-    src_prefix = here.parent / 'labextension'
-
-    if not src_prefix.exists():
-        src_prefix = Path(sys.prefix) / f'share/jupyter/labextensions/{labext_name}'
-
-    return [{'src': str(src_prefix), 'dest': labext_name}]
+    return [
+        {
+            'src': 'labextension',
+            'dest': '@jupyter-widgets/jupyterlab-manager'
+        }
+    ]
```

Comment:
Simplifying this way is closer [to the cookiecutter](https://github.com/jupyter-widgets/widget-ts-cookiecutter/blob/master/%7B%7Bcookiecutter.github_project_name%7D%7D/%7B%7Bcookiecutter.python_package_name%7D%7D/__init__.py#L23) and works well with editable AND non-editable installs.

The old approach was not working well with a non-editable install.